docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -65,7 +65,7 @@
         {{- end }}
```

Comment:
If possible, could we add a top-level comment at the beginning of the file explaining what this layout does + its prerequisites for use? For the latter, I mean any files / state that it expects to be present in order to function correctly (like the new generated yml files you added to the API docs build.